containers/wordpress/wp_opt/Dockerfile

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files
+# (the "Software"), to deal in the Software without restriction,
+# including without limitation the rights to use, copy, modify, merge,
+# publish, distribute, sublicense, and/or sell copies of the Software,
+# and to permit persons to whom the Software is furnished to do so, 
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+# OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
+# OR OTHER DEALINGS IN THE SOFTWARE.
+#
+# SPDX-License-Identifier: MIT
+ARG baseimage=wp_base
+FROM $baseimage
+LABEL authors="nitin.tekchandani@intel.com,lin.a.yang@intel.com,uttam.c.pawar@intel.com"
+
+COPY files/perf.fdata /home/${USERNAME}/oss-performance
+
+# Optimization: Set environment variable to have zend use large pages
+ENV USE_ZEND_ALLOC_HUGE_PAGES=1 
+
+# Build php 7.4.16 for bolting
+RUN sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 7 && \
+    sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-7 7 && \
```

Comment:
At the end of a line /usr/bin/g++-7 7 && \, is "7" a typo on line 33 and 34?

Reply:
i don't believe so but I can recheck the research i did to enable this.